packages/flutter/lib/src/widgets/_web_image_web.dart

```diff
@@ -366,3 +390,20 @@ class RenderWebImage extends RenderShiftedBox {
     );
   }
 }
+
+extension on BoxFit {
```

Comment:
Extensions are not something we allow in the framework, this is in the style guide here: https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-using-extension

Reply:
Especially since this can easily be replaced by a global function. :)